### DISCUSSION TITLE
How can I add a line break from the frontmatter in mdx?

### DISCUSSION THREAD

**Discussion by JulSeb42** [vercel/next.js#24055] [Help]
Hey,

I'm working on a project where on each MDX page the intro is written in the frontmatter. I managed to display it, but I can't add line breaks. How can I do it?

Or another solution would be to make a component with the content of this section, but it needs to take some content from the frontmatter too. How could I make the component taking both children and these frontmatter content?

Thanks!

**Comment by michrome** [comment]
Assuming your frontmatter is YAML, then I think you’d need to use a [literal style with block chomping](https://yaml-multiline.info/). Something like this:

```yaml
example: |-
  Several lines of text,
  with some "quotes" of various 'types',
  and also a blank line:
  
  plus another line at the end.
```

**Reply by JulSeb42** [reply]
That's exactly how it's written but the line breaks don't work

**Reply by tpiersall-reifyhealth** [reply]
Does this work using MDX for anyone else?  It's not working for me (producing a line break)
